active/RadioButtonsAPI.md

```diff
@@ -0,0 +1,105 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a class extension of ListView that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective list of RadioButtons using RadioButtons.
+
+```xaml
+<RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Dark</x:String>
+    <x:String>Light</x:String>
+    <x:String>Windows Default</x:String>         
+</RadioButtons>
+```
+
+# API Details
+
+### RadioButtons
+
+```
+namespace Microsoft.UI.Xaml.Controls
+{
+[webhosthidden]
+[contentproperty("Items")]
+unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
+{
+    RadioButtons();
+
+    Object ItemsSource;
+    Windows.Foundation.Collections.IVector<Object> Items{ get; };
+    Windows.UI.Xaml.DataTemplate ItemTemplate;
```

Comment:
Needs ItemTemplateSelector to complete the trinity?

Reply:
I'm not sure what you mean... ItemTemplateSelector should be in the IDL or in the API Notes?

Reply:
Right ... it's missing. It can wait until when @teaP is back but there's a couple APIs I'd expect to see in this control that I don't.

Reply:
I was trying to keep it light and simple. Why would this control ever need multiple ItemTemplates?